SocialCareCaseViewerApi/V1/Gateways/IDatabaseGateway.cs

```diff
@@ -15,7 +15,7 @@ public interface IDatabaseGateway
         CreateAllocationResponse CreateAllocation(CreateAllocationRequest request);
         string GetPersonIdByNCReference(string nfReference);
         string GetNCReferenceByPersonId(string personId);
-        List<Worker> GetWorkers(int teamId);
+        List<Worker> GetWorkers(int? teamId, int? id);
```

Comment:
Please update this method signature so it matches the one used in the updated implementation